src/security/src/test/kotlin/SodiumTests.kt

```diff
@@ -0,0 +1,18 @@
+import org.junit.Test
+import org.vaccineimpact.api.security.SodiumPasswordEncoder
+
+class SodiumTests {
+    @Test
+    fun `hashes password`() {
+
+        val encoder = SodiumPasswordEncoder()
+        val testPassword = "this is a password"
+
+        val hashedPw = encoder.encode(testPassword)
+        val decodedPw = encoder.matches(testPassword, hashedPw)
+
+        assert(decodedPw)
+
+    }
+
+}
```

Comment:
This doesn't match the project code style - braces, etc. 

Also that final condition is confusing - `encoder.matches` returns a boolean, so the variable shouldn't be called `decodedPw`.

Maybe rewrite as: `assertThat(encoder.matches(testPassword, hashedPw)).isTrue()`